README.md

```diff
@@ -948,6 +948,41 @@ assert out.model.name == "my-model"
 assert out.model.uuid == state_in.model.uuid
 ```
 
+## CloudSpec
+
+You can set CloudSpec information in the state.
```

Comment:
I think this is a very good suggestion. I will move them up.

I will put "for example" in another line, so that users won't think the example below is one only with the required parameters.

As a side note, there are two similar issues in the same README.md examples where mandatory fields detail appear below the code. (Secrets, Relation). Since it's not related to CloudSpec, I will not touch them in this PR.